tests/unit/utils/regexp-test.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import regexp from 'dummy/utils/regexp';
```

Comment:
Just a note: if you remove the export of the utilities from the `app` tree, this will have to be updated to import from the addon, too.